### ISSUE TITLE
Show source improvement

### ISSUE THREAD

**Issue by niyonx** [yohanchatelain/pytracer#13]
When the show source is enabled, an improvement to the code console would be to display the line directly on change, without the user having to scroll and look for the line.

**Comment by yohanchatelain** [comment]
Yes it would be great. But I don't know how to do it with dash-ace. Do you know?

**Comment by niyonx** [comment]
Nope, I don't know as well, I thought I could write a javascript snippet for it but it seems not that simple. I created an issue in the dash-ace repo in the meantime.

**Comment by yohanchatelain** [comment]
Resolved in Pytracer 2.0: built-in source code context viewer in the explorer tab.
